archives.groovy

```diff
@@ -87,7 +87,7 @@ def updateMonitors() {
 }
 
 def requestUpdate() {
-  sparkRequest = '''curl --verbose -X POST http://@@HOST@@:7077/v1/submissions/create --header "Content-Type:application/json;charset=UTF-8" --data '{
+  sparkRequest = '''curl --verbose --max-time <10000> -X POST http://@@HOST@@:7077/v1/submissions/create --header "Content-Type:application/json;charset=UTF-8" --data '{
```

Comment:
suggest to remove the "<" and ">" . The submit / status update commands should be pretty short lived, so, a time-out of 30 seconds should be sufficient. You should be able to test in your mac terminal.